R/board_gdrive.R

```diff
@@ -171,10 +171,13 @@ required_pkgs.pins_board_gdrive <- function(x, ...) {
 
 # Helpers -----------------------------------------------------------------
 
+possibly_drive_ls <- function(...) {
+  purrr::possibly(googledrive::drive_ls)(...)
+}
```

Comment:
If you're going to write a wrapper, I think you might as well just inline `possibly()`, i.e.

```suggestion
possibly_drive_ls <- function(...) {
  tryCatch(googledrive::drive_ls(...), error = function(err) NULL)
}
```